src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1743,12 +1751,9 @@ public String toString() {
 			if (isNotBlank(getMatrixCoefficients())) {
 				result.append(", Matrix Coefficients: ").append(getMatrixCoefficients());
 			}
-			if (isNotBlank(avcLevel)) {
-				result.append(", AVC Level: ").append(getAvcLevel());
+			if (isNotBlank(getVideoFormatProfile())) {
+				result.append(", Video Format Profile: ").append(getVideoFormatProfile());
```

Comment:
You appelation totally confuse me...
_Video Coding Level_ should be more important to log than the profile, that is the first part before the @